tests/app/notifications/test_process_notification.py

```diff
@@ -339,3 +339,35 @@ def test_persist_notification_with_international_info_does_not_store_for_email(
     assert persisted_notification.international is False
     assert persisted_notification.phone_prefix is None
     assert persisted_notification.rate_multiplier is None
+
+
+@pytest.mark.parametrize('recipient, expected_recipient_normalised', [
+    ('7900900123', '447900900123'),
+    ('+447900   900 123', '447900900123'),
+    ('  07700900222', '447700900222'),
+    ('07700900222', '447700900222'),
+    (' 73122345678', '73122345678'),
+    ('360623400400', '360623400400')
```

Comment:
Can you add a test with brackets and dashes just to make sure we’re stripping them out?